app/src/main/java/org/scottishtecharmy/soundscape/screens/home/locationDetails/LocationDetailsScreen.kt

```diff
@@ -64,35 +80,112 @@ fun LocationDetailsScreen(
 
 @Composable
 fun LocationDetails(
-                    locationDescription : LocationDescription,
-                    onNavigateUp: () -> Unit,
-                    latitude: Double?,
-                    longitude: Double?,
-                    heading: Float,
-                    createBeacon: (latitude: Double, longitude: Double) -> Unit,
-                    enableStreetPreview: (latitude: Double, longitude: Double) -> Unit,
-                    modifier: Modifier = Modifier) {
+    locationDescription: LocationDescription,
+    onNavigateUp: () -> Unit,
+    latitude: Double?,
+    longitude: Double?,
+    heading: Float,
+    createBeacon: (latitude: Double, longitude: Double) -> Unit,
+    enableStreetPreview: (latitude: Double, longitude: Double) -> Unit,
+    modifier: Modifier = Modifier
+) {
 
     Column(
-        modifier = modifier
-            .fillMaxHeight(),
+        modifier = modifier.fillMaxHeight(),
     ) {
         CustomAppBar(
-            title =  stringResource(R.string.location_detail_title_default),
+            title = stringResource(R.string.location_detail_title_default),
             onNavigateUp = onNavigateUp,
         )
-        Text(
-            text = locationDescription.title,
-            modifier = Modifier.padding(top = 20.dp, bottom = 5.dp),
-            style = MaterialTheme.typography.bodyMedium,
-            color = MaterialTheme.colorScheme.surfaceBright
-        )
-        Text(
-            text = "A DESCRIPTION BASED ON THE TILE DATA!",
-            modifier = Modifier.padding(top = 20.dp, bottom = 5.dp),
-            style = MaterialTheme.typography.bodyMedium,
-            color = MaterialTheme.colorScheme.surfaceBright
-        )
+        Column(
+            modifier = Modifier
+                .padding(horizontal = 15.dp, vertical = 20.dp)
+                .verticalScroll(rememberScrollState()),
+            verticalArrangement = Arrangement.spacedBy(20.dp)
+        ) {
+            Column(
+                verticalArrangement = Arrangement.spacedBy(15.dp)
+            ) {
+                locationDescription.name?.let {
+                    Text(
+                        text = it,
+                        style = MaterialTheme.typography.titleLarge,
+                        color = Color.White
+                    )
+                }
+                locationDescription.distance?.let {
+                    Row(
+                        verticalAlignment = Alignment.CenterVertically,
+                        horizontalArrangement = Arrangement.spacedBy(10.dp)
+                    ) {
+                        Icon(
+                            imageVector = Icons.Filled.Map,
+                            contentDescription = "map",
+                            tint = Color.Yellow
```

Comment:
seems to be color Foreground2 in Color.kt
maybe we can set this color as the tertiary color or our theme and then use 

```suggestion
                        color = MaterialTheme.colorScheme.tertiary
```